conda_smithy/ci_skeleton.py

```diff
@@ -37,18 +36,16 @@ def _insert_into_gitignore(
 ):
     """Places gitignore contents into gitignore."""
     # get current contents
-    fname = os.path.join(feedstock_directory, ".gitignore")
-    print("Updating " + fname)
-    if os.path.isfile(fname):
+    fname = Path(feedstock_directory, ".gitignore")
+    print("Updating ", fname.name)
```

Comment:
```suggestion
    print("Updating", fname.name)
```

Reply:
I am not sure, isn't `.name` going to give you only the _file_ name (not the whole path as previously)?

I think it should be `print("Updating", fname)`.